### DISCUSSION TITLE
Parallax

### DISCUSSION THREAD

**Discussion by DonGeilo** [redeclipse/base#1135] [Maps and Content Showcase]
![parallax](https://user-images.githubusercontent.com/37516698/37702911-50d17260-2cf4-11e8-9b19-28e6cf704e0c.jpg)

[parallax.zip](https://github.com/red-eclipse/world/files/1832682/parallax.zip)

Hey Guys,
I made a map called Parallax. The Layout works well with all modes. It doesnt feature Multi though...
By now, its only compatible with 1.6, but i want to make it work in 2.0.

Can someone explain me how to get the dev version, so i can work on it?

If I learn how to do the lights in the new version, maybe I can be helpful in porting old maps. We'll see...

**Comment by IceflowRE** [comment]
looks cool!
And: https://redeclipse.net/wiki/Obtain_development_version

**Comment by qreeves** [comment]
The layout looks nice and tight, if a little square at times, but I think the texturing could use some work. Quite a few of the ones you've used have actually been removed in the dev version because they're so "meh". We're focusing on *quality* over *quantity* now.

We really do need more people porting what *good* maps we already have. It isn't just about re-lighting them, but also an opportunity to freshen up the texturing and layout.

**Comment by DonGeilo** [comment]
Ty for your feedback.
Yeah, there are lots of textures I was was wondering why they are even existing. I think its a good choice to reduce them!
I can see the problem with porting the maps. Im not sure if I am experienced enough though... I will try to get into the topic and see if I can help.

**Comment by cloudziq** [comment]
You are also a welcome in coop editing ;p  (if we manage to be in the same time), feel free to ask about every little thing, so nothing will be problematic for you
